brix-core/src/main/java/org/brixcms/jcr/RepositoryUtil.java

```diff
@@ -29,6 +29,33 @@
 public class RepositoryUtil {
     private static final Logger logger = LoggerFactory.getLogger(Brix.class);
 
+    /**
+     * register a brix basic node as a unstructured node type, reason is: accoring to JCR spec 2,
+     * a mixin may only have other mixins as supertypes
+     *
+     * @param workspace
+     */
+    public static void registerBrixUnstructuredMixin(Workspace workspace) {
+        String cnd = "[brix:unstructured] mixin" +
+                "  - * (undefined) multiple " +
+                "  - * (undefined) " +
+                "  + * (nt:base) sns version ";
+
+        NodeTypeManager manager = null;
```

Comment:
How the `try` could become bigger if you add the type of the class before the variable name ?
I.e. instead of `manager = ....` it will become `NodeTypeManager manager = ...`